### PR TITLE
JAMES-1993 Add limitations to the migration documentation

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by aduprat** [comment]
Rebased of #740

File: src/site/xdoc/server/upgrade-2.3.xml [hunk 1]
```diff
@@ -65,6 +65,9 @@
      
      <p>The migration will be provided and fully supported in next release.</p>
      
+     <p>We advise you to use external mail migration tools such as
```
Comment (rouazana) [inline src/site/xdoc/server/upgrade-2.3.xml]: "If you can't wait for the next release, ..."

File: src/site/xdoc/server/upgrade-2.3.xml [hunk 2]
```diff
@@ -65,7 +65,7 @@
      
      <p>The migration will be provided and fully supported in next release.</p>
      
-     <p>We advise you to use external mail migration tools such as 
+     <p>If you can't wait for the next release, xe advise you to use external mail migration tools such as
```
Comment (rouazana) [inline src/site/xdoc/server/upgrade-2.3.xml]: s/xe/we/ :)

Comment (aduprat) [inline src/site/xdoc/server/upgrade-2.3.xml]: WTF

**Pull request merged**